eth/tracers/js/internal/tracers/evmdis_tracer.js

```diff
@@ -71,7 +71,7 @@
 				opinfo["ops"] = [];
 				this.stack.push(opinfo);
 				break;
-			case "RETURN":
+			case "RETURN": case "REVERT":
```

Comment:
This tracer is also using `SUICIDE`, as opposed to `SELFDESTRUCT` -- I think this tracer is kind of bitrotted. Also, it was written to be used along with `disasm` (https://github.com/Arachnid/evmdis ) which hasn't been updated since 2018.

Reply:
Yeah I was going to submit a PR renaming SUICIDE in multiple places, but decided to put a stop to creating new PRs for now 😅

Reply:
Fixed this instance of suicide here.